stack/stack_noaa.yaml

```diff
@@ -36,7 +36,7 @@ udunits:
 hdf5:
   build: YES
   version: 1.10.6
-  shared: NO
+  shared: YES
```

Comment:
Please do not make NOAA libraries shared

Reply:
This setting includes shared libraries without removing the static libraries. This is needed to build MAPL at this time, but I'll work with NASA to remove this requirement.

Reply:
It turns out this requires a minor change in the MAPL build system. I've submitted an issue and will revert this change once a new MAPL version is released.

Reply:
This issue is caused by an incomplete `NetCDF::NetCDF_Fortran` static target created by `FindNetCDF.cmake` in `CmakeModules`. This target only provides `netcdff.a`, not the full list of required libraries. See issue [#51](https://github.com/NOAA-EMC/CMakeModules/issues/51).

@aerorahul, @kgerheiser - could this issue be overcome with custom environment settings?

Reply:
The following setting works on Hera and it does not need additional shared libraries:
```
-DNetCDF_Fortran_EXTRA_LIBRARIES="`nc-config --libs`"
```
Please let me know if this change is acceptable.

Reply:
I think that's fine.

Reply:
Done. Please let me know if there are additional issues in regard to CI.